comixed-frontend/src/app/comic-import/effects/selected-comic-files.effects.ts

```diff
@@ -0,0 +1,37 @@
+/*
+ * ComiXed - A digital comic book library management application.
+ * Copyright (C) 2020, The ComiXed Project
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program. If not, see <http:/www.gnu.org/licenses>
+ */
+
+import { Injectable } from '@angular/core';
+import { Actions, createEffect, ofType } from '@ngrx/effects';
+
+import { concatMap } from 'rxjs/operators';
+import { EMPTY } from 'rxjs';
+
+import * as SelectedComicFilesActions from '../actions/selected-comic-files.actions';
+
+@Injectable()
+export class SelectedComicFilesEffects {
+  constructor(private actions$: Actions) {}
+
+  loadSelectedComicFiless$ = createEffect(() => {
+    return this.actions$.pipe(
+      ofType(SelectedComicFilesActions.clearComicFileSelections),
```

Comment:
Couldn't we import only the clearCOmicFIleSelections action?
import { clearComicFileSelections } from '../actions/selected-comic-files.actions';
And use it : 
ofType(clearComicFileSelections),

Reply:
Yes. I've fixed it and updated the PR.